src/Core/Lock/Capabilities/ICanLock.php

```diff
@@ -19,7 +19,7 @@
  *
  */
 
-namespace Friendica\Core\Lock\Capability;
+namespace Friendica\Core\Lock\Capabilities;
```

Comment:
Why the change? What does the plural give you that the shorter singular didn't? I don't think anyone is assuming that there's only one capability in the Capability namespace.

Reply:
hmm .. yes, you're right

I just wasn't sure if changing it from singular to plural or vice versa :D

Reply:
Just go for the simpler term. 😄